scripts/postgresql-setup.sh

```diff
@@ -171,10 +178,19 @@ function restore_snapshot {
 	  fi
 	tmp_dir=$(mktemp "${directory}" -t db-sync-snapshot-XXXXXXXXXX)
 	tar -zxvf "$1" --directory "$tmp_dir"
-	db_file=$(find "$tmp_dir/" -iname "*.sql")
-	lstate_file=$(find "${tmp_dir}/" -iname "*.lstate")
-	mv "${lstate_file}" "$2"
-	psql --dbname="${PGDATABASE}" -f "${db_file}"
+	if test -d "${tmp_dir}/db/" ; then
+	  # New pg_dump format
+      lstate_file=$(find "${tmp_dir}/" -iname "*.lstate")
+	  mv "${lstate_file}" "$2"
+	  drop_db
+	  pg_restore --jobs="${numcores}" --format=directory --dbname=cexplorer "${tmp_dir}/db/"
+	else
+	  # Old pg_dump format
+	  db_file=$(find "$tmp_dir/" -iname "*.sql")
```

Comment:
Is this a typo ?

Reply:
In the comment? Yes.

Changed to 
```
# Old snapshot format produced by this script
```

Reply:
I was more curious about db_file=$(find "$tmp_dir/" -iname "*.sql")  VS.  lstate_file=$(find "${tmp_dir}/" -iname "*.lstate")

the missing {}